Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js

```diff
@@ -562,8 +562,16 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
             this.service.removeNavigationItem(identifier, zone);
 
             Y.Array.each(this.service.get(zone + "NavigationItems"), function (item) {
+                var itemIdentifier;
+
+                if (item.config) {
+                    itemIdentifier = item.config.identifier;
+                } else {
+                    itemIdentifier = item.identifier; //subtreeItem
```

Comment:
weird patch, I think in this case item is an instance of eZ.NavigationItemSubtreeView so to get it's identifier you should write `item.get('identifier')`